src/components/material/MaterialAvailabilityText/physical/MaterialAvailabilityTextPhysical.tsx

```diff
@@ -21,7 +21,8 @@ const MaterialAvailabilityTextPhysical: React.FC<
   const faustIds = convertPostIdsToFaustIds(pids);
   const { data, isLoading, isError } = useGetHoldings({
     faustIds,
-    config
+    config,
+    blacklist: "availability"
```

Comment:
:+1: This previously used the default value `false` for `useAvailabilityBlacklist`. This was mapped to the `pickup` type. Now it is the `availability` type. This is as requested in the ticket.